### ISSUE TITLE
acl: permissions=rX always marked as changed

### ISSUE THREAD

**Issue by drzraf** [ansible/ansible-modules-core#2151]
Bug report
Ansible 1.9.3
Debian Jessie

Requesting a X permission (uppercase X) is always marked as changed.

Step to reproduce:

```
$ getfacl /tmp/file.php
# file: file.php
# owner: admin
# group: admin
user::rw-
user:www-data:r-x       #effective:r--
group::r-x          #effective:r--
group:operator:rwx      #effective:rw-
mask::rw-
other::---
```

```
ansible localhost -Cvs -m acl -a "name=/tmp/file.php entity=www-data etype=user permissions=rX default=no state=present"
```

Expected result:
`"change": false`

Actual result:

```
host | success >> {
    "acl": [
        "user::rw-",
        "user:www-data:r-x",
        "group::r-x",
        "group:operator:rwx",
        "mask::rwx",
        "other::---"
    ],
    "changed": true,
    "msg": "user:www-data:r-X is present"
}
```

**Comment by astorije** [comment]
Please follow the issue template described here: https://github.com/ansible/ansible/blob/devel/CONTRIBUTING.md

In particular, Ansible version is pretty relevant here...

**Comment by drzraf** [comment]
seems fixed in 2.0.1.0
can't reproduce
